index.html

```diff
@@ -1,41 +1,191 @@
 <!DOCTYPE html>
-
 <html lang="en">
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>My Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter&family=Roboto:wght@100&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="/css/style.css">
```

Comment:
There is a problem when loading your CSS Stylesheet. I opened  the file in Firefox and Chrome and the stylesheet doesn't load. Consider the following change:

C
```suggestion
    <link rel="stylesheet" href="css/style.css">
```